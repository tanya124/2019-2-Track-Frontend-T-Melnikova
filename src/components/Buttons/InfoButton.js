import React from 'react';
import styled from '@emotion/styled';
import list from '../../assets/list.svg';

const Button = styled.img`
	width: 3em;
	margin-right: 10px;
	margin-left: 20px;
	&:hover {
		filter: invert(1);
	}
`;


function InfoButton() {
	return (
		<Button src={ list } />
	);
}

export default InfoButton;