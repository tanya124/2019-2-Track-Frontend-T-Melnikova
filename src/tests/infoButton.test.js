import React from 'react'
import renderer from 'react-test-renderer'
import InfoButton from '../components/Buttons/InfoButton'

it('renders correctly', () => {
  const tree = renderer.create(<InfoButton />).toJSON()
  expect(tree).toMatchSnapshot()
})
