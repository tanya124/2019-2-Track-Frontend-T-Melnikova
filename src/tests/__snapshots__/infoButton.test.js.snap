// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<img
  className="css-thc3d8"
  src="list.svg"
/>
`;
